tests/functional/vault/test_strategies.py

```diff
@@ -68,7 +68,12 @@ def test_liquidation_after_hack(chain, gov, vault, token, TestStrategy):
 def strategy_with_wrong_vault(gov, token, vault, Vault, TestStrategy):
     otherVault = gov.deploy(Vault)
     otherVault.initialize(
-        token, gov, gov, token.symbol() + " yVault", "yv" + token.symbol(), gov,
+        token,
+        gov,
+        gov,
+        token.symbol() + " yVault",
+        "yv" + token.symbol(),
+        gov,
```

Comment:
Looks like some `black` changes snuck in here?

Reply:
Yes, would like this to get merged first https://github.com/yearn/yearn-vaults/pull/310